package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gnosis.pm/gp-v1",
-  "version": "2.1.0-rc.0",
+  "version": "2.1.0-rc.17",
```

Comment:
17? LoL

Reply:
Yes!

I was commenting most of the cy.yml and making looots of tests :)